### PR TITLE
feat(components/pages): add recently accessed service support

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Review by Blackbaud-SteveBrush — DISMISSED** [review]

File: libs/components/router/src/lib/modules/recently-accessed/recently-accessed.service.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+import { Observable, of } from 'rxjs';
```
Comment (johnhwhite) [inline libs/components/router/src/lib/modules/recently-accessed/recently-accessed.service.ts]: `of` is not used and should not be here.

**Review by johnhwhite — COMMENTED** [review]
Looks good. One small thing.

File: libs/components/router/src/lib/modules/recently-accessed/recently-accessed.service.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply libs/components/router/src/lib/modules/recently-accessed/recently-accessed.service.ts]: Ah, we didn't turn on `noUnusedLocals` in `5.x.x`.

Comment (Blackbaud-PaulCrowder) [reply libs/components/router/src/lib/modules/recently-accessed/recently-accessed.service.ts]: Good catch! Fixed.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**